docs/source/Publications.rst

```diff
@@ -0,0 +1,8 @@
+Publications
+============
+
+*SyReC* is academic software. Thus, many of its built-in algorithms have been published as scientific papers.
+
+If you use *SyReC* in your work, we would appreciate if you cited :cite:labelpar:`wille2019towardsHDLsynthesis`.
+
+.. bibliography::
```

Comment:
This bibliography is way too empty. At least include all the relevant papers for the methods that are currently implemented.

Reply:
I will add this accordingly in the following commits.

Reply:
done 👍🏻